build.sbt

```diff
@@ -59,6 +63,8 @@ lazy val chronoscala = (project in file("."))
           }
         }
       },
+      mimaBinaryIssueFilters +=
+        ProblemFilters.exclude[DirectMissingMethodProblem]("jp.ne.opt.chronoscala.Interval.apply"),
```

Comment:
Could you explain what's the intention of this line?

As far as I tested on local, the test passes without this line.

Reply:
Only ```sbt ";++2.12.10;test"``` fails.
But I can't explain why. :cry:.

Reply:
Hmm.
I did javap for `jp/ne/opt/chronoscala/Interval.class` in `chronoscala_2.12-0.3.0.jar`, and found that `apply(Object, Object)` is generated which is not exist in `chronoscala_2.12-0.3.2.jar`.

```
   #28 = Utf8               apply
   #29 = Utf8               (Ljava/lang/Object;Ljava/lang/Object;)Ljava/lang/Object;
```
I'm suspecting this is related to https://github.com/scala/bug/issues/11207

Reply:
Yeah, the issue is likely due to that Scala compiler bug.

- That bug is appeared in Scala 2.12.7 and fixed in 2.12.8.
- And I confirmed that chronoscala 0.3.0 is compiled by 2.12.7, and 0.3.1 or later are compiled by 2.12.8.

So,
- could you limit the exclusion filter to only when scalaBinaryVersion == 2.12, and version == 0.3.0 ?
- and please add a comment about "the exclusion filter is necessary for Scala compiler bug" ?